src/main/groovy/edu/oregonstate/mist/trailsapi/resources/TrailResources.groovy

```diff
@@ -81,4 +81,25 @@ public class TrailsResource extends Resource {
 	Boolean trailValidator(Trail trail) {
 		trail.name && trail.zipCode && trail.difficulty
 	}
+
+	@GET
+	@Produces(MediaType.APPLICATION_JSON)
```

Comment:
You shouldn't need this annotation on every method if you use the same annotation at the class level.